names.py

```diff
@@ -0,0 +1,8 @@
+test_database_name = "test3"
+database_name = "evil_car"
+roads = "roads"
+violations_collection = "violations"
+testingCollection = "testingCollection"
+starting_longitude = "startingLongitude"
+starting_latitude = "startingLatitude"
+obstacles_collection = "obstacles" 
```

Comment:
what is the use of those ?

they same to be fields names in mongodb document  or collections names
Am i write? if so
can you write the structure of every document as comment here

or better than commenting
i suggest using dict for those names so obstacles document for example would be
```python
obstacles_collection = {name = "obstacles",
                                      long: 'long',
                                      lat: 'lat',
                                      time: 'time'}
```
then we can keep track of both the document structure and its fields names as well collection name

Reply:
this a was a **lazy ,string-less, loosely coupled** way to make a globally used string because, you know, no one loves string typos, though there might be a better way to do so, i don't know. 
Using dicts will violate the string-less rule and i want to avoid string whenever possible.

Reply:
but i still think we need a better way to hold those values (cllection-names and fields-key) 
what about class with static properties
```python
from enum import Enum
class obstacles_collection(Enum) 
                                      @property: 
                                      def name():
                                             return "obstacles
                                      
                                      @property: 
                                      def long():
                                             return "long"

.
. 
.
. so on

 ```
so for example send object function would be
```python
from names import obstacles_collection
def send_obstacles():
 data_manager.add_data(obstacles_collection.name,
                                        {obstacles_collection.long: request.args.get('long'),
                                         obstacles_collection.lat: request.args.get('lat'),
                                         obstacles_collection.time: str(datetime.datetime.now())})
```

instead of 
```python
def send_obstacles():
 data_manager.add_data(names.obstacles_collection, {'long': request.args.get('long'),
                                                       'lat': request.args.get('lat'),
                                                       'time': str(datetime.datetime.now())})
```

or for better practice I suggest using odm for monogdb 
https://github.com/mongodb/pymodm

may add it in another pull request
i will add this as an issue __suggestion__

Reply:
https://github.com/abdo1819/server_evilcar/issues/4